build-samples.yml

```diff
@@ -23,7 +23,7 @@ steps:
       Set-PSDebug -Trace 1
       Write-Host "${{ parameters.path }}"
 
-      & $env:msbuildpath "${{ parameters.path }}" /r /p:Configuration=Release /p:WasmShellMonoRuntimeExecutionMode=Interpreter /p:AotAssemblies=false /p:PublishTrimmed=false /p:WasmShellILLinkerEnabled=false /ds "/bl:$(build.artifactstagingdirectory)\$(Agent.JobName).binlog" ;
+      & $env:msbuildpath "${{ parameters.path }}" /r /p:Configuration=Release /p:WasmShellMonoRuntimeExecutionMode=Interpreter /p:AotAssemblies=false /p:PublishTrimmed=false /p:WasmShellILLinkerEnabled=false /p:RunAOTCompilation=false /ds "/bl:$(build.artifactstagingdirectory)\$(Agent.JobName).binlog" ;
```

Comment:
AotAssemblies is deprecated. You can only keep `RunAOTCompilation` that you added and remove `/p:AotAssemblies=false`